_i18n/messages.properties

```
@@ -1,3 +1,4 @@
 UnableToDownloadAttachmentScanStatusNotClean=Unable to download the attachment as scan status is not clean.
 AttachmentSizeExceeded=File size limit exceeded beyond {0}.
 MultiUpdateNotSupported=Multi update is not supported.
+AttachmentMimeTypeDisallowed=The attachment mime type '{mimeType}' is not allowed.
```

Isn't mime type a little bit too technical?

```suggestion
AttachmentMimeTypeDisallowed=The attachment file type '{mimeType}' is not allowed.
```